addons/website_livechat/views/website_visitor_views.xml

```diff
@@ -41,8 +41,10 @@
                     Speaking With
                     <div name="livechat_operator_id"
                          class="o_field_many2one_avatar o_field_widget font-weight-bold float-right d-inline-block">
-                        <img t-attf-src="/web/image/res.partner/#{record.livechat_operator_id.raw_value}/avatar_128"
-                             alt="Operator Avatar" class="o_m2o_avatar"/>
+                        <div class="o_m2o_avatar d-inline-block">
+                            <img t-attf-src="/web/image/res.partner/#{record.livechat_operator_id.raw_value}/avatar_128"
+                                alt="Operator Avatar"/>
+                        </div>
```

Comment:
Hello @std-odoo 

Code is a bit lighter indeed.
However, with the current implementation, I'm getting all the css from the regular widget:

![image](https://user-images.githubusercontent.com/41282536/129211261-535aa75d-2bc3-427c-b86f-368647946e9e.png)

Which may be a bit more robust than manually specifying classes like rounded-circle etc, in case they differ for mobile display for example (I don't think they actually do but well... you see what I mean :D ).
Also, there are some minor differences (19x19 for example).

What do you think?

Reply:
I do not really like using the class `o_m2o_avatar` because it's only used in `web` (for the widget) and `website_livechat`... Same for the class `o_field_many2one_avatar` (so the JS team might rename the class one day maybe and break this view).

But if you prefer your version I do not have a big issue with it and you can green :)